lib/particles/accelerated_particle.dart

```diff
@@ -41,7 +40,7 @@ class AcceleratedParticle extends CurvedParticle with SingleChildParticle {
   @override
   void update(double t) {
     speed += acceleration * t;
-    position += speed * t - (acceleration * pow(t, 2)) / 2;
+    position += speed * t - (acceleration * t * t) / 2;
```

Comment:
so `pow` is declared for `num` instead of `int`/`double` and strong mode doesn't like it. probably dart needs to do some updates so it's compatible but for this I think multiplication is perfectly fine. other option is `pow(a, b).toDouble()`.